ui/app/AppLayouts/Communities/popups/CommunityMemberMessagesPopup.qml

```diff
@@ -0,0 +1,144 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+import QtGraphicalEffects 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import utils 1.0
+import shared.views.chat 1.0
+
+StatusDialog {
+    id: root
+
+    property var store
+    property var chatCommunitySectionModule
+    property var memberMessagesModel: chatCommunitySectionModule.memberMessagesModel
+    property string memberPubKey: ""
+    property string displayName: ""
+
+
+    width: 800
+    padding: 0
```

Comment:
I believe if recommendation is : 

> Note: Different styles may specify the default padding for certain controls in different ways, and these ways may change over time as the design guidelines that the style is based on evolve. To ensure that these changes don't affect the padding values you have specified, it is best to use the most specific properties available. For example, rather than setting the padding property:
>
> padding: 0
>
> set each specific property instead:
>
> leftPadding: 0
rightPadding: 0
topPadding: 0
bottomPadding: 0